src/shared/components/palette/dynamic-list/write-dynamic-list-field.html

```diff
@@ -0,0 +1,16 @@
+<div class="form-group" [ngClass]="{'form-group-error': !dynamicListFormControl.valid && dynamicListFormControl.touched}">
+
+    <label [for]="id()">
+      <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
+      <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+      <span class="error-message" *ngIf="dynamicListFormControl.errors && dynamicListFormControl.touched">{{dynamicListFormControl.errors | ccdFirstError}}</span>
+    </label>
+  
+    <select class="form-control ccd-dropdown bottom-30" [id]="id()" 
+    [(ngModel)]="caseField.value" [formControl]="dynamicListFormControl">
+      <option [ngValue]=null>--Select a value--</option>
+      <option [ngValue]="type.code" *ngFor="let type of caseField.list_items">{{type.label}}</option>
+    </select>
+  
+  </div>
+  
```

Comment:
Getting a warning about no newline at end of file here...

Reply:
done